configure.ac

```diff
@@ -168,6 +169,62 @@ AS_IF([test "x$enable_exp_cq" == xyes],
 	])
 )
 
+#
+# VMAPOLL
+#
+AC_ARG_ENABLE(
+    [vmapoll],
+    AC_HELP_STRING(
+        [--enable-vmapoll],
+        [Enable alternative API to standard sockets API for receiving packets (default=no)]))
+AC_MSG_CHECKING(
```

Comment:
Detailed explanation should be added to User Manual. Configure help is just 1 summary line.